INSTALL.rst

```diff
@@ -1,156 +1,230 @@
+worked, run the following in a Python shell or Jupyter notebook:
+
+.. code-block:: python
+
+  import skimage
+  print(skimage.__version__)
+
+or, from the command line
+
+.. code-block:: sh
+
+   python -c "import skimage; print(skimage.__version__)"
+
+(Try ``python3`` if ``python`` is unsuccessful.)
+
+You'll see the version number if ``scikit-image`` is installed and
+an error message otherwise.
+
+Scientific Python distributions
+------------------------------------------------------------------------------
+
+In a single install these give you Python,
+``scikit-image`` and libraries it depends on, and other useful scientific
+packages. They install into an isolated environment, so they won't conflict
+with any existing installed programs.
+
+Drawbacks are that the install can be large and you may not get
+the most recent ``scikit-image``.
+
+We recommend one of these distributions:
+
+- `Anaconda <https://www.anaconda.com/distribution/>`_
+- `Python(x,y) <https://python-xy.github.io/>`_
+- `WinPython <https://winpython.github.io/>`_
+
+When using the ``scikit-image``
+documentation, make sure it's for the version you've installed (see
+`Version check <#version-check>`_ above).
+
+
+Installation via pip and conda
+------------------------------------------------------------------------------
+
+These install only ``scikit-image`` and its dependencies; pip has an option to
+include related packages.
+
+.. _install-via-pip:
+
+pip
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Prerequisites to a pip install: You're able to use your system's command line to
+install packages and are using a
+`virtual environment \
+<https://towardsdatascience.com/virtual-environments-104c62d48c54?gi=2532aa12906#ee81>`_
+(any of
+`several \
+<https://stackoverflow.com/questions/41573587/what-is-the-difference-between-venv-pyvenv-pyenv-virtualenv-virtualenvwrappe>`_\
+).
+
+Nothing will stop you from using pip without a virtual environment,
+but `it's a bad idea that will bite you later \
+<https://en.wikipedia.org/wiki/Dependency_hell>`_.
 
-We are assuming that you have default Python environment already configured on
-your computer and you intend to install ``scikit-image`` inside of it. If you
-want to create and work with Python virtual environments, please follow the
-instructions on `venv`_ and `virtual environments`_.
+To install the current ``scikit-image`` you'll need at least Python 3.6. If
+your Python is older, pip will find the most recent compatible version.
 
-There are two ways you can install ``scikit-image`` on your preferred Python
-environment.
+.. code-block:: sh
 
-1. Standard Installation
-2. Development Installation
+  # Update pip
+  python -m pip install -U pip
+  # Install scikit-image
+  python -m pip install -U scikit-image
 
-1. Standard Installation:
--------------------------
+To include a selection of other scientific python packages as well,
```

Comment:
To also install related packages that expand scikit-image's capabilities to include, e.g., parallel processing, replace the last line with

Reply:
Please feel free to make this change.